packages/app/src/cli/services/local-storage.ts

```diff
@@ -13,6 +13,7 @@ export interface CachedAppInfo {
   storeFqdn?: string
   updateURLs?: boolean
   previousAppId?: string
+  apiKey?: string
 }
```

Comment:
Added this, but I noticed we may use the `appId` field to store the `apiKey`, one example [here](https://github.com/Shopify/cli/blob/8da87af1592efb9cffc565fd9299e3903aaa6746/packages/app/src/cli/services/context.ts#L248-L254).  But I opted just to add `apiKey` as an optional type.